script/validate/dco

```diff
@@ -1,12 +0,0 @@
-#!/usr/bin/env bash
-
-set -eu -o pipefail
-
-if ! command -v git-validation; then
-    >&2 echo "ERROR: git-validation not found. Install with:"
-    >&2 echo "    go install github.com/vbatts/git-validation@latest"
-    exit 1
-fi
-
-verbosity="${DCO_VERBOSITY--v}"
-GIT_CHECK_EXCLUDE="./vendor:./script/validate/template" git-validation "$verbosity" -range "$COMMIT_RANGE" -run DCO,short-subject,dangling-whitespace
```

Comment:
The old script was also validating for dangling whitespaces in the commit messages, and for a maximum subject length; do we have a replacement for that?

Reply:
Added it back the sandboxed way so we are align with the rest of our validation process. See the respective commit.

Reply:
Wondering now; if we keep the script, should we keep the DCO in the dockerfile check as well (to allow a local validate)?

Not a blocker for me, but wondering if it's worth keeping, as it's only a extra check in that stage now

Reply:
Don't think we need a local validation for DCO imo.